### PR TITLE
Zhaoyu/deleted files

### PR DESCRIPTION
**Pull request by DongZhaoYu**
list the deleted files and show warning in log

### CODE REVIEW

File: src/cleaner/scripts/check_deleted_files.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    files = common.run_cmd("source ./scripts/list_deleted_files.sh 2>/dev/null", log)
+    if len(files) <= 1:
+        log.info("no deleted files found.")
+        return
+    else:
+        # skip the field names from the command
+        files = files[1:]
+
+    for f in files:
+        f_fields = f.split(" ")
```
Comment (xudifsd) [inline src/cleaner/scripts/check_deleted_files.py]: Is output of the first column guaranteed not having space?

File: src/cleaner/scripts/check_deleted_files.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import multiprocessing
+from cleaner.utils import common
+
+logger = multiprocessing.get_logger()
+
+
+def list_and_check_files(arg, log=logger):
+    files = common.run_cmd("source ./scripts/list_deleted_files.sh 2>/dev/null", log)
```
Comment (xudifsd) [inline src/cleaner/scripts/check_deleted_files.py]: If the script is not too complicated, call command directly would be more readable and efficient(without calling bash)

File: src/cleaner/scripts/check_deleted_files.py (hunk 1 above)
Comment (DongZhaoYu) [reply src/cleaner/scripts/check_deleted_files.py]: Yes, the first column of the output is process id and it has no space.

File: src/cleaner/scripts/check_deleted_files.py (hunk 2 above)
Comment (DongZhaoYu) [reply src/cleaner/scripts/check_deleted_files.py]: Consider the performance. It is reasonable to call it directly. Will change it.

**Pull request merged**